src/msw/spinctrl.cpp

```diff
@@ -385,17 +385,7 @@ bool wxSpinCtrl::Create(wxWindow *parent,
                    name, size.x, bestSize.GetWidth());
     }
```

Comment:
`sizeCtrl` is unused now, right? So all this code could basically be replaced with just
```cpp
if ( size.x > 0 && size.x < GetBestSize().x )
{
     wxLogDebug(...);
}
```
couldn't it?